static/bidder-info/huaweiads.yaml

```diff
@@ -1,7 +1,7 @@
 endpoint: "https://acd.op.hicloud.com/ppsadx/getResult"
 disabled: true
 maintainer:
-  email: hwads@huawei.com
+  email: prebid@huawei.com

```

Comment:
@ahmetfaruk59 We have sent a verification email to `prebid@huawei.com`. Please reply to the verification mail with "received" message to complete the verification process. 
Thank you!

Reply:
received

Reply:
Verification completed!

Reply:
Thanks a lot!